packages/analytics/src/index.ts

```diff
@@ -1,5 +1,6 @@
 /**
- * Firebase Analytics
+ * The Firebase Analytics Web SDK.
+ * This SDK currently does not work in Node.js.
```

Comment:
remove 'currently' to align with the other documentation. Also it makes it sound like it WILL work in node.js but are we ready to make that commitment.